_data/simple-icons.json

```diff
@@ -12997,6 +12997,11 @@
             "hex": "217DC6",
             "source": "https://seeklogo.com/vector-logo/184137/renren-inc"
         },
+        {
+            "title": "Replicate",
+            "hex": "141414",
```

Comment:
Charlie from Replicate here—this can be `#000000`. Thanks so much for creating this, @sebastien46. 😊